src/Apps/Artwork/Components/PricingContext.tsx

```diff
@@ -23,9 +24,12 @@ function PricingContext({ artwork }: PricingContextProps) {
       <Sans size="2" weight="medium">
         Price
       </Sans>
-      <Sans size="2">
-        Price ranges of {artwork.pricingContext.filterDescription}
-      </Sans>
+      <Flex>
+        <Sans size="2">
+          Price ranges of {artwork.pricingContext.filterDescription}
+        </Sans>
+        <PricingContextModal />
```

Comment:
Could you add a test for the presence of this in the PricingContext test file? :pray: